src/pages/ProductDesc/ProductDesc.js

```diff
@@ -1,8 +1,56 @@
 import React from "react";
 import "./ProductDesc.scss";
+import { useState, useEffect } from "react";
+import RecommendProducts from "./components/RecommendProducts/RecommendProducts";
+import NewProducts from "./components/NewProducts/NewProducts";
+import CartAll from "./components/CartAll/CartAll";
 
 const ProductDesc = () => {
-  return <div>ProductDesc입니다.</div>;
+  const [productDatas, setProductData] = useState([]);
+
+  const [sizeModal, setSizeModal] = useState(false);
```

Comment:
boolean naming convention에 맞게 is 붙여주세요!
```suggestion
  const [isSizeModal, setIsSizeModal] = useState(false);
```